api/utils/config.cjs

```diff
@@ -55,3 +55,6 @@ function requireArgs(required) {
 		);
 	}
 }
+
+module.exports.CLIENT_SECRET = process.env.GITHUB_CLIENT_SECRET;
+module.exports.CLIENT_ID = process.env.GITHUB_CLIENT_ID;
```

Comment:
I don't think this is the correct way to include these values. We already have an existing module.exports line

Better would be to have 

```
github: {
  GITHUB_CLIENT_SECRET: process.env.GITHUB_CLIENT_SECRET,
  // etc.
}
```

and add this to the extsing config object